src/v2/guide/class-and-style.md

```diff
@@ -179,16 +179,16 @@ data: {
 }
 ```
 
-Again, the object syntax is often used in conjunction with computed properties that return objects.
```

Comment:
Discussion : 

Retirer le `à` et ne pas tenir compte de `to` car en français, avec le `à` on a tendence à ce demander « lier quoi à ça » ce qui fait que du coup la phrase manque d'un sujet (le contenu de v-bind en l'occurence).
> C'est souvent une bonne idée de lier directement ~à~ un objet de style, pour que le template soit plus propre

On peut aussi rajouter le sujet mais ça me plait moins car là on « invente » du texte (même si on garde le sens)

> C'est souvent une bonne idée de lier **`v-bind:style`** directement à un objet de style, pour que le template soit plus propre